### PR TITLE
new fixes for orientation and WA for focus

### PR DESCRIPTION
**Pull request by gkdeepa**

### CODE REVIEW

**Review by xihua-chen — APPROVED** [review]

**Review by shivasku82 — APPROVED** [review]

**Review by swaroopbalan — APPROVED** [review]

File: bsp_diff/caas_cfc/vendor/intel/external/project-celadon/camera-vhal/0001-VHAL-changes-for-orien-w.r.t-facing.patch [hunk 1]
```diff
@@ -0,0 +1,4346 @@
...
+diff --git a/src/CameraSocketServerThread.cpp b/src/CameraSocketServerThread.cpp
+index 62d4f25..770b8a8 100644
+--- a/src/CameraSocketServerThread.cpp
++++ b/src/CameraSocketServerThread.cpp
+@@ -15,11 +15,11 @@
+  */
+ //#define LOG_NDEBUG 0
+ //#define LOG_NNDEBUG 0
+-#define LOG_TAG "CameraSocketServerThread: "
++#define LOG_TAG "CameraSocketServerThread: Deepa"
```
Comment (shyjumon-n) [inline bsp_diff/caas_cfc/vendor/intel/external/project-celadon/camera-vhal/0001-VHAL-changes-for-orien-w.r.t-facing.patch:981]: I think we shall remove these debug prints ?

File: bsp_diff/caas_cfc/vendor/intel/external/project-celadon/camera-vhal/0001-VHAL-changes-for-orien-w.r.t-facing.patch [hunk 2]
```diff
@@ -0,0 +1,4346 @@
...
+@@ -64,9 +80,19 @@ CameraSocketServerThread::CameraSocketServerThread(std::string suffix,
+ #endif
+     std::string sock_path = "/ipc/camera-socket" + suffix;
+     char *k8s_env_value = getenv("K8S_ENV");
+-    mSocketPath = (k8s_env_value != NULL && !strcmp(k8s_env_value, "true"))
+-            ? "/conn/camera-socket" : sock_path.c_str();
+-    ALOGI("%s camera socket server path is %s", __FUNCTION__, mSocketPath.c_str());
++    mSocketPath = (k8s_env_value != NULL && !strcmp(k8s_env_value, "true")) ? "/conn/camera-socket"
++                                                                            : sock_path.c_str();
++    ALOGI("%s Deepa camera socket server path is %s", __FUNCTION__, mSocketPath.c_str());
```
Comment (shyjumon-n) [inline bsp_diff/caas_cfc/vendor/intel/external/project-celadon/camera-vhal/0001-VHAL-changes-for-orien-w.r.t-facing.patch:1023]: We shall remove these debug prints

**Review by shyjumon-n — CHANGES_REQUESTED** [review]
Please follow the comments !

**Review by shyjumon-n — APPROVED** [review]
Looks good to me

**Pull request merged**